src/sage/features/interfaces.py

```diff
@@ -108,6 +108,23 @@ def __classcall__(cls):
         return InterfaceFeature.__classcall__(cls, 'mathics', 'sage.interfaces.mathics')
 
 
+class Regina(InterfaceFeature):
+    r"""
+    A :class:`~sage.features.Feature` describing whether :class:`sage.interfaces.regina.Regina`
+    is present and functional.
+
+    EXAMPLES::
+
+        sage: from sage.features.interfaces import Regina
+        sage: Regina().is_present()  # not tested
+        FeatureTestResult('regina', False)
+    """
+
+    @staticmethod
+    def __classcall__(cls):
+        return InterfaceFeature.__classcall__(cls, 'regina', 'sage.interfaces.regina')
```

Comment:
```suggestion
    @staticmethod
    def __classcall__(cls):
        return InterfaceFeature.__classcall__(cls, 'regina', 'sage.interfaces.regina')
```
Forgive my strictness, but this needs a doctest.